docs: Adapt Potential duplicates to new API [DHIS2-17501] (#1395)
dhis2/dhis2-docs: src/developer/web-api/tracker.md

}
```

## Potential Duplicates  

Potential duplicates are records identified by the data deduplication feature as possibly being duplicates. Due to the nature of this feature, the API endpoint has certain restrictions.

A potential duplicate represents a pair of records suspected to be duplicates.

To retrieve a list of potential duplicates, use the following endpoint:

    GET /api/potentialDuplicates

The response payload for a potential duplicate looks like this:

```json
{
  "created": "2024-06-04T10:11:29.110",
  "lastUpdated": "2024-06-04T10:11:29.110",

>   "original": "<id>",

Reviewer: In all tracker docs we are using `UID` instead of `ID` where it makes sense.
When we talk about tracked entity we should always use UID.

Then, it makes sense to leave the `id` to identify the `potentialDuplicate` itself

  "duplicate": "<UID>",
  "status": "OPEN|INVALID|MERGED",
  "id": "<id>"
}
```

These are the parameters this endpoint accepts:

| Parameter name | Description | Type | Allowed values |
|---|---|---|---|

> | trackedEntities | List of tracked entities | List of string (separated by comma)| existing tracked entity IDs |

Reviewer: ```suggestion
| trackedEntities | List of tracked entities | List of string (separated by comma)| existing tracked entity UIDs |
```

| status | Potential duplicate status | string | `OPEN`, `INVALID`, `MERGED`, `ALL` |

To inspect individual potential duplicate records, use the following endpoint:

    GET /api/potentialDuplicates/<id>

To create a new potential duplicate, use this endpoint:

    POST /api/potentialDuplicates

> The payload you provide must include the IDs of the original and duplicate tracked entities. New potential duplicates are open by default.

Reviewer: ```suggestion
The payload you provide must include the UIDs of the original and duplicate tracked entities. New potential duplicates are open by default.
```

```json
{
  "original": "<UID>",
  "duplicate": "<UID>"
}
```

| Status code | Description
|---|---|

> | 400 | Input original or duplicate is null or has invalid id

Reviewer: ```suggestion
| 400 | Input original or duplicate is null or has invalid uid
```

| 403 | User do not have access to read origianl or duplicate TEs
| 404 | TE not found
| 409 | Pair of original and duplicate TEs already existing

To update the status of a potential duplicate, use the following endpoint:

    PUT /api/potentialDuplicates/<id>

| Parameter name | Description | Type | Allowed values |
|---|---|---|---|
| status | Potential duplicate status | string | `OPEN`, `INVALID` |

| Status code | Description
|---|---|
| 400 | You can't update a potential duplicate to MERGED as this is possible only by a merging request
| 400 | You can't update a potential duplicate that is already in a MERGED status

### Merging Tracked Entities
Tracked entities can be merged together if they are deemed viable. To initiate a merge, the first step is to define two tracked entities as a Potential Duplicate. The merge endpoint moves data from the duplicate tracked entity to the original tracked entity and deletes the remaining data of the duplicate.

> To merge a Potential Duplicate, or the two tracked entities the Potential Duplicate represents, use the following endpoint:

Reviewer: ```suggestion
To merge a Potential Duplicate, i.e. the two tracked entities the Potential Duplicate represents, use the following endpoint:
```
